async-query/src/test/java/org/opensearch/sql/spark/asyncquery/AsyncQueryExecutorServiceSpec.java

```diff
@@ -137,7 +136,7 @@ public class AsyncQueryExecutorServiceSpec extends OpenSearchIntegTestCase {
 
   @Override
   protected Collection<Class<? extends Plugin>> nodePlugins() {
-    return Arrays.asList(TestSettingPlugin.class);
+    return List.of(TestSettingPlugin.class);
```

Comment:
can u use static import for the `List.of` method ?

Reply:
I think in this case, we need to import all of `java.util.List` because `List` is used elsewhere in the class.

There are a few cases in the code bases where we could switch to using a static imports as below. Would you want me to do so? My only concern is that `of` is pretty generic, in terms of readability?

```
import static java.util.List.of;
...
return of(TestSettingPlugin.class);
```